tests/test_subpackages.py

```diff
@@ -77,3 +78,12 @@ def test_hash_includes_recipe_files(testing_workdir, testing_config):
     a given output."""
     recipe = os.path.join(subpackage_dir, 'script_install_files')
     outputs = api.build(recipe, config=testing_config)
+
+
+@pytest.mark.skipif(sys.platform == "win32",
```

Comment:
I think you can allow it on win32 if you put in posix as a build-time requirement.